guides/source/active_record_validations.md

```diff
@@ -593,52 +642,43 @@ which specifies the value must be an instance of `Numeric` and attempts to parse
 the value if it is a `String`.
 
 NOTE: By default, `numericality` doesn't allow `nil` values. You can use
-`allow_nil: true` option to permit it. Notice that for `Integer` and `Float`
+`allow_nil: true` option to permit it. For `Integer` and `Float`
 columns empty strings are converted to `nil`.
 
 The default error message when no options are specified is _"is not a number"_.
 
 There are also many options that can be used to add constraints to acceptable
 values:
 
-* `:greater_than` - Specifies the value must be greater than the supplied
-  value. The default error message for this option is _"must be greater than
-  %{count}"_.
-* `:greater_than_or_equal_to` - Specifies the value must be greater than or
-  equal to the supplied value. The default error message for this option is
-  _"must be greater than or equal to %{count}"_.
-* `:equal_to` - Specifies the value must be equal to the supplied value. The
-  default error message for this option is _"must be equal to %{count}"_.
-* `:less_than` - Specifies the value must be less than the supplied value. The
-  default error message for this option is _"must be less than %{count}"_.
-* `:less_than_or_equal_to` - Specifies the value must be less than or equal to
-  the supplied value. The default error message for this option is _"must be
-  less than or equal to %{count}"_.
-* `:other_than` - Specifies the value must be other than the supplied value.
-  The default error message for this option is _"must be other than %{count}"_.
-* `:in` - Specifies the value must be in the supplied range.
-  The default error message for this option is _"must be in %{count}"_.
-* `:odd` - Specifies the value must be an odd number. The default error message
-  for this option is _"must be odd"_.
-* `:even` - Specifies the value must be an even number. The default error
-  message for this option is _"must be even"_.
+| Option                      | Description                                                              | Default Error Message                       |
+| --------------------------- | ------------------------------------------------------------------------ | ------------------------------------------- |
```

Comment:
This table repeats the information above. Would it make sense to point the reader to the other section by linking it here?

Reply:
I'm assuming you're referring to the `comparison` helper table, but please correct me if I'm wrong. I think that since they're different helpers that can be read independently and are not related, I'd prefer to keep them separate. In addition, this table in `numericality` has a few more options than the one in `comparison`